.github/workflows/ios.yml

```diff
@@ -18,8 +18,8 @@ jobs:
         env:
           scheme: ${{ 'SDKHostApp' }}
           platform: ${{ 'iOS Simulator' }}
-          os: ${{ '16.2' }}
-          device: ${{ 'iPhone 13' }}
+          os: ${{ '18.1' }}
+          device: ${{ 'iPhone 16' }}
```

Comment:
I think we can just delete these. What happens if you don't include them?

Reply:
These params are used in the `xcodebuild` invocation build later in the script so that command will fail. Instead, we should update the `xcodebuild` command to use whatever the latest os version is. Can you make that update?